cli/cmd/main.go

```diff
@@ -890,21 +890,22 @@ func createDataFilter(c *cli.Context) (*datapb.Filter, error) {
 		filter.MimeType = c.StringSlice(dataFlagMimeTypes)
 	}
 	if c.StringSlice(dataFlagTags) != nil {
-		tagsLen := len(c.StringSlice(dataFlagTags))
-		switch c.StringSlice(dataFlagTags)[0] {
-		case "tagged":
-			if tagsLen == 1 {
+		if len(c.StringSlice(dataFlagTags)) == 1 {
```

Comment:
I think this can be simplified with:
```
if len == 1 && [0] == tagged {
   // all tagged case
} else if len == 1 && [0] == untagged {
 // all untagged case
} else {
 // or case (the current else case)
}
```

Reply:
yes you're right! ty